### PR TITLE
Allow anonymous browsing of public buckets.

### PR DESCRIPTION
**Pull request by krishnasrinivas**

### CODE REVIEW

File: app/js/reducers.js [hunk 1]
```diff
@@ -134,6 +134,10 @@ export default (state = {buckets:[], visibleBuckets:[], objects:[], storageInfo:
     case actions.SET_SHARE_OBJECT:
       newState.shareObject = Object.assign({}, action.shareObject)
       break
+    case actions.SET_PREFIX_WRITABLE:
+      newState.prefixWritable = action.prefixWritable
+      console.log(action.prefixWritable)
```
Comment (harshavardhana) [inline app/js/reducers.js]: you can remove this if not needed?

**Comment by harshavardhana** [comment]
Do you see this issue locally? 

```
npm ERR! peerinvalid The package react@0.14.8 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer react-copy-to-clipboard@4.2.3 wants react@^0.14 || ^15
npm ERR! peerinvalid Peer react-custom-scrollbars@2.3.0 wants react@^0.14.5
npm ERR! peerinvalid Peer react-dom@0.14.8 wants react@^0.14.8
npm ERR! peerinvalid Peer react-dropzone@3.9.0 wants react@^0.14.0 || ^15.0.0
npm ERR! peerinvalid Peer react-onclickout@2.0.5 wants react@^15.x
npm ERR! peerinvalid Peer react-addons-test-utils@0.14.8 wants react@^0.14.8
npm ERR! peerinvalid Peer react-redux@4.4.6 wants react@^0.14.0 || ^15.0.0-0 || ^15.4.0-0
npm ERR! peerinvalid Peer react-router@2.8.1 wants react@^0.14.0 || ^15.0.0
0mnpm ERR! peerinvalid Peer react-bootstrap@0.28.5 wants react@>=0.14.0
```

**Comment by harshavardhana** [comment]
Let me just disable travis for miniobrowser.

**Comment by krishnasrinivas** [comment]
> npm ERR! peerinvalid The package react@0.14.8 does not satisfy its siblings' peerDependencies requirements!
> npm ERR! peerinvalid Peer react-copy-to-clipboard@4.2.3 wants react@^0.14 || ^15
> npm ERR! peerinvalid Peer react-custom-scrollbars@2.3.0 wants react@^0.14.5
> npm ERR! peerinvalid Peer react-dom@0.14.8 wants react@^0.14.8
> npm ERR! peerinvalid Peer react-dropzone@3.9.0 wants react@^0.14.0 || ^15.0.0
> npm ERR! peerinvalid Peer react-onclickout@2.0.5 wants react@^15.x
> npm ERR! peerinvalid Peer react-addons-test-utils@0.14.8 wants react@^0.14.8
> npm ERR! peerinvalid Peer react-redux@4.4.6 wants react@^0.14.0 || ^15.0.0-0 || ^15.4.0-0
> npm ERR! peerinvalid Peer react-router@2.8.1 wants react@^0.14.0 || ^15.0.0
> 0mnpm ERR! peerinvalid Peer react-bootstrap@0.28.5 wants react@>=0.14.0

i don't see these errors, do you see it only on travis?

**Comment by harshavardhana** [comment]
Only travis i think @krishnasrinivas

**Comment by krishnasrinivas** [comment]
let me check build process on a fresh codebase

**Pull request merged**